README.md

```diff
@@ -1,57 +1,73 @@
-Portfolio
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-    My portfolio in 2021
+# Project Name
+  Portfolio
+> My portfolio in 2021. To be updated.
 
-image
+![screenshot](https://github.com/Somdotta07/Portfolio/blob/Feature/media/screenshot.PNG)
 
+It's a portfolio project and template taken from Figma
 
-The portfolio is made of HTML and CSS . To be updated.
-Built With
+## Built With
 
-    HTML and CSS
+- HTML and CSS
+- NA
+- NA
```

Comment:
- [ ] Since some of the items here are Not Applicable you should leave them blank. So you could have something like:
```
- HTML
- CSS
```
only